src/dom/svg/SVGForeignObjectElement.js

```diff
@@ -0,0 +1,11 @@
+// @ts-check
+
+import { SVGAnimatedLength } from './SVGAnimatedLength.js'
+import { SVGGraphicsElement } from './SVGGraphicsElement.js'
+
+export class SVGForeignObjectElement extends SVGGraphicsElement {
+  x = new SVGAnimatedLength(self, 'x')
```

Comment:
shouldnt that be "this"?

Reply:
Ooh, of course! Trusted `// @ts-check` too much, I guess.